Procfile

```
@@ -1,5 +1,7 @@
 release: python ./manage.py migrate
-web: gunicorn --bind 0.0.0.0:$PORT dandiapi.wsgi --timeout 25
+# The Django application uses a custom middleware to set the username in the request context by setting the X-Request-Username header.
+# We use that here to make gunicorn log the username in the access logs.
+web: gunicorn --bind 0.0.0.0:$PORT dandiapi.wsgi --timeout 25 --access-logformat='%(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s" username:%({x-request-username}o)s'
```

Perhaps we could delineate the username more clearly?
```suggestion
web: gunicorn --bind 0.0.0.0:$PORT dandiapi.wsgi --timeout 25 --access-logformat='%(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s" <username:%({x-request-username}o)s>'
```

Previously I had suggested placing the username at the beginning of the log message, however, that does shift the contents depending on the username length, and I think wrapping the username in the `< >` characters makes it readable enough, without the downsides.